packages/admin-ui/ui/src/i18n/index.ts

```diff
@@ -10,6 +10,7 @@ void i18n
   // https://www.i18next.com/overview/configuration-options
   .init({
     fallbackLng: ["en"],
+    supportedLngs: ["en", "es"],
```

Comment:
suggestion: not sure it should be part of that pr

Reply:
todo: It should not be part of the PR.

Reply:
Ouch, that's right, not sure what happened there.